datastore/google/cloud/datastore/_onestore_v3.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto2";
```

Comment:
It's mostly based on GAE's actual usage of the onestore proto, the name is a bit misleading. We can rename this as `gae_key.proto`.

Also, It's *very* unlikely that GAE will ever produce a change that breaks all previously encoded keys.